test-at/pom.xml

```diff
@@ -11,6 +11,17 @@
 
     <artifactId>test-at</artifactId>
 
+    <repositories>
+        <repository>
+            <id>sonatype-oss-snapshot</id>
```

Comment:
Later we can upload the snapshots to internal nexus and we can skip this repo. But for now I think this is perfect :)

Only one comment. Do we want daily updates?